base/applications/mspaint/history.cpp

```diff
@@ -201,6 +203,7 @@ void ImageModel::Crop(int nWidth, int nHeight, int nOffsetX, int nOffsetY)
     if (!hbmNew)
     {
         ShowOutOfMemory();
+        ClearHistory();
```

Comment:
Note: there are two other `ShowOutOfMemory()` calls in main.cpp and one in dib.cpp.
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ShowOutOfMemory
Question: Should you attempt to call `ClearHistory()` for these as well, or isn't it necessary? (I have no idea about that)

Reply:
Fixed.